From 9cf648227bec649f0dbdc28985b59b32d679c5c3 Mon Sep 17 00:00:00 2001
From: Mingkai Hu <mingkai.hu@nxp.com>
Date: Mon, 18 Jul 2016 18:08:01 +0800
Subject: [PATCH 009/124] arm64: dts: ls2088a: Add 'dma-coherent' for PCI nodes

The 'dma-coherent' indicates that the hardware IP block can ensure
the coherency of the data transferred from/to the IP block. This
can avoid the software cache flush/invalid actions, and improve
the performance significantly.

Signed-off-by: Minghuan Lian <Minghuan.Lian@nxp.com>
Signed-off-by: Mingkai Hu <mingkai.hu@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
index d197af6..f9af24d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls2088a.dtsi
@@ -447,6 +447,7 @@
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
+		dma-coherent;
 		fsl,lut_diff;
 		num-lanes = <4>;
 		bus-range = <0x0 0xff>;
@@ -472,6 +473,7 @@
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
+		dma-coherent;
 		fsl,lut_diff;
 		num-lanes = <4>;
 		bus-range = <0x0 0xff>;
@@ -497,6 +499,7 @@
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
+		dma-coherent;
 		fsl,lut_diff;
 		num-lanes = <8>;
 		bus-range = <0x0 0xff>;
@@ -522,6 +525,7 @@
 		#address-cells = <3>;
 		#size-cells = <2>;
 		device_type = "pci";
+		dma-coherent;
 		fsl,lut_diff;
 		num-lanes = <4>;
 		bus-range = <0x0 0xff>;
-- 
2.1.0.27.g96db324

